source/seismogram/data-format.rst

```diff
@@ -0,0 +1,57 @@
+波形数据格式
+============
+
+日常科研中，我们会接触不同的波形数据文件格式，最常见的是 SAC、SEED、以及 miniSEED。
+
+SAC
+---
+
+SAC 格式由 `SAC <https://ds.iris.edu/ds/nodes/dmc/software/downloads/SAC/>`__
+（Seismic Analysis Codes）软件定义，但许多其他软件也支持该格式。每个 SAC 文件包含头段区
+和时间序列数据区，其中头段区定义了与台站、事件、走时、以及用户自定义信息等相关的变量。
+
+该格式有利于数据处理和分析，因此是日常科研中常用的数据格式。
+但是，若选择该格式来存储数据，则需要很大的磁盘空间。
+
+SEED
+----
+
+SEED 格式全称 Standard for the Exchange of Earthquake Data（地震数据交换标准），
+主要用于地震学时间序列数据和相关元数据的归档和交换。最新的 SEED 标准是 2012 年发布的
+`SEED 2.4 <http://www.fdsn.org/pdf/SEEDManual_V2.4.pdf>`__。
+
+虽然该格式包含了完备的地震学观测数据，但是不利于处理和分析波形数据，
+同时在存储和交换时存在一些冗余的信息。近年来，一些时间序列数据格式（如 miniSEED）
+和元数据格式（如 `stationXML <https://www.fdsn.org/xml/station/>`__）
+的组合正在逐渐代替 SEED 格式。
```

Comment:
rdseed is no longer maintained (see https://ds.iris.edu/ds/nodes/dmc/manuals/rdseed/), is end of life.

Besides rdseed, I don't know any other tools that can read SEED format. Does it mean that the SEED format itself is close to being "dead"?

Reply:
Or a related question, do we want to add a tutorial for rdseed?

Reply:
> rdseed is no longer maintained (see https://ds.iris.edu/ds/nodes/dmc/manuals/rdseed/), is end of life.

Great to kown.

> Besides rdseed, I don't know any other tools that can read SEED format.

From ObsPy documentation and its email list, maybe read() and io.xseed and read the data and the metadata parts of the full seed, respectively.

- https://docs.obspy.org/packages/obspy.io.mseed.html
- https://docs.obspy.org/master/packages/obspy.io.xseed.html
- http://lists.swapbytes.de/archives/obspy-users/2016-August/002099.html
- http://lists.swapbytes.de/archives/obspy-users/2013-February/000877.html

> Does it mean that the SEED format itself is close to being "dead"?

Even we may use ObsPy or rdseed to read full SEED, I think SEED will not be used any more.

> Or a related question, do we want to add a tutorial for rdseed?

My answer is NO.

Perhaps, we can move SEED to miniseed subsection to only present sac and miniseed. I'd like to remove the links to rdseed and sac2mseed since we may never use them in the future.

Reply:
Do we want to add ASDF format? or we can add it later when we really use it.

Reply:
> > rdseed is no longer maintained (see https://ds.iris.edu/ds/nodes/dmc/manuals/rdseed/), is end of life.
> 
> Great to kown.

A related news: [Retirement of full SEED Data Volumes from IRIS DMC](http://www.iris.washington.edu/ds/newsletter/vol21/no1/509/retirement-of-full-seed-data-volumes-from-iris-dmc/)

> From ObsPy documentation and its email list, maybe read() and io.xseed and read the data and the metadata parts of the full seed, respectively.

Need to give it a try.

> Do we want to add ASDF format? or we can add it later when we really use it.

Not now.

Reply:
I think we can have a subsection for "Other formats", and list a few other commonly seen formats, but don't explain too much.

Reply:
> > From ObsPy documentation and its email list, maybe read() and io.xseed and read the data and the metadata parts of the full seed, respectively.
> 
> Need to give it a try.

I tyied it. ObsPy can read data and metadata from full seed, although there is a warning when reading data part using read().